simple-mpc-query.el

```diff
@@ -5,6 +5,7 @@
 ;; Author: Joren Van Onder <joren.vanonder@gmail.com>
 ;; Maintainer: Joren Van Onder <joren.vanonder@gmail.com>
 ;; Keywords: multimedia, mpd, mpc
+;; Package-Requires: ((emacs "24.4") (s "1.10.0"))
```

Comment:
Any reason why you added `(emacs "24.4")`? Not saying it's wrong, just not sure why it's needed :).

Reply:
Ah, because I _think_ `s` requires at least emacs 24. Also, because, well, emacs 24 seems to be safe-ish enough to consider "old" :-)